scripts/postprocess-osx/packagebuilding-howto

```diff
@@ -1,7 +1,7 @@
 Steps tp create a OSX package:
 
 1) Build LibreCAD
-2) Test of LibreCAD is working, make sure resources are included like fonts, helpiles and templates
+2) Test of LibreCAD is working, make sure resources are included like fonts, helfiles and templates
```

Comment:
@arpnakumari, all modifications you apply to this branch will show up here. Thus you must not delete your pull request. Work on it and when it is fine we can merge.